### ISSUE TITLE
Vamist's Neutral nerfs A.K.A Add Slavery Back Dammit

### ISSUE THREAD

**Issue by Jim-Smiff** [TFlippy/kag_territorycontrol#158]
Not an issue, but something that I think should be said. Vamist’s neutral nerfs are not a bad thing. When you look into Faction VS Neutral fights, neutrals will always have the upperhand. Their advantage? The fact that they have nothing to lose. If their base gets blown up, they can just simply buy a book for a couple hundred coins, a gun, some ammo, maybe bombs. Then throw themselves at the faction until they die or the faction dies. If the neutral dies, they just do the same thing, new base, new weapons. They lost nothing. While factions always have something to lose, their base. Not only that, factions are static. Neutrals can base wherever they please. Sure, factions may be able to build multiple faction bases, but it requires lots of resources. Also, you're spreading out your area you need to defend, and unlike cocok, who can “be in two places at once”, most people can't. So you end up not being able to defend your bases. So, the 50% coin drop, fixed the issue of neutrals just being able to make new bases right after death, and buying guns. So, stop complaining about the coin drop. 

But, I will complain about this, slavery gone. The best way to stop a neutral, or anyone of that matter is to just slave them. Yeah, the slavery system needs to be a bit improved, but without it, neutrals can just throw themselves at the factions without worry. Also you can’t make the Mithrios altar. Which is technically an issue.

**Comment by asperger-sind** [comment]
I agree with the slavery part.

I disagree with the neutral part a lot.
A strong faction with 2-3 players who can hold their weight and some more can fight neutrals forever without a loss due to various factories and strategies that don't require them to go out of their way and lose half their capital.

For a quick example so the players and players who don't play the game so often like Vamist understand.

Me and Superhemfi start a faction base and in the first 10 minutes of it's existence got a valid factory with a constant stream of resources and some sort of protection that can aid us.

In the meantime we already get some enemies as neutrals and they will try to attack us as soon as they get gear (5-6 minutes)
In that time we can already get valid bomber, bomb, hoob/juggernaut and other various strong ways of attacking

(bomber is an oversight lol but yeah we still can counter than in a second or two)

Even if a neutral loses that, they can keep attacking, yes.
But by the time they attack again our defense and offence will be 3 times as strong and the damage from the neutral will be gone.

Sadly not everyone can counter that and I hope that big bombs will ease it somehow for neutrals and both factions but the question remains still.

Are neutrals op? No, if you are good enough to counter them then they are a pushover that is overshadowed by factions.

Can a neutral settle into a faction after a loss to get the same advantages?
No. The neutral faction will get obliterated since if the neutral lost with ok or even good gear, then the faction can hold off against threats that are much more than a simple neutral faction.

(especially with contesting like holy shit that garbage isn't fair, i hate playing against it and i hate playing with it)

In the end if a faction of good players settles in for at least 10 minutes and start snowballing then it's just not possible to counter.
If the faction gets dug up before they can do anything then it might be a valid counter, but they become neutrals and as you said they can make a new base anywhere, at any time and maybe even be ready to counter whatever killed them, if they're good enough that is.

big wall of text hihi have a fun time reading this jim or whoever else reads this

**Comment by Jim-Smiff** [comment]
yeah, I did not take skill into account. But yes, factions can overpower neutrals if the faction is good at the game, but the same can be said about neutrals. From what I've seen, in most long faction vs neutral wars (when the players we equal skill), the neutrals win more often than not, because the neutrals just keep throwing themselves at the faction until the faction runs out of resources. This is the issue, and it was fixed by Vamist changing the money drop rate. But with slavery gone there is no way to "stop" a netral. But yea, kinda a bit dumb on my part for forgetting to account for skil. And yes, funni text wall. I had fun reading it.

**Comment by asperger-sind** [comment]
From what I've seen factions fighting neutrals of equal skill always win since they just have more options.
For example.

KOS and absolute bloodshed around the map, disallowing the neutral(s) while a teammate is getting you all the resources you need to continue and makes defenses.

Properly placed automatic defenses and some attention will make 90% of the sieges futile.

And don't even get me started on megafactions.
If a faction has 4-5 players and they are good then it's the end of the world for any sort of neutral that they for some reason don't eradicate the second they spawn.

Basically a 2 man faction with 1 managing the base and resources and the other one taking care of pesky neutrals or factions can be unstoppable if they are good enough.

After all I want to say this:
Neutrals are not as strong as a faction because a faction has the tools to overpower the neutrals and then just keep their strength fixed so they can't do anything about it. If a megafaction assembles and with actually good players then as I said, it is the end for all things neutral.

I think the coin drop rate should be changed to force newer players to actually get good and use a thing called "brain" during play. It will make the stronger players more strong, but you can't really fix that with any buff, can you?

Slavery should be added back, A slaved person will die in 5 minutes after enslavement. It will combat many issues regarding neutrals, and make fighting neutrals not that pointless.

**Comment by Jim-Smiff** [comment]
Huh, weird. Most the fights I've seen end with neutrals winning. But yea. A new slavery system needs to be in the game. TC would not be TC without one. Also slavery is the only way of stopping neutrals without destroying the entire map. Which, from my experience no one enjoys playing on a radiated land of void, unbreakable blocks, and dirt back walls. Because, as of now, the only way to stop a neutral is by stripping them of all their resources. So ye, please, add slavery back.

**Comment by asperger-sind** [comment]
Neutrals don't destroy the entire map, factions do.

**Comment by Jim-Smiff** [comment]
I poorly worded my phrasing, that was the point I was trying to make. Factions have to resort to destroying the map to stop neutrals. That's why slavery is needed back.
